modules/loader/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/dsl/Transaction.scala

```diff
@@ -62,6 +62,11 @@ trait Transaction[F[_], C[_]] {
    */
   def run[A](io: C[A]): F[A]
 
+  /**
+   * Same as run, but narrowed down to transaction to allow migration error handling.
+   */
+  def run_(io: C[Unit]): F[Unit]
```

Comment:
Would it better to name it something more descriptive ? `runWithErrorHandler` maybe ? `run` and `run_` can be easily mixed up I think.

Reply:
That naming convention is used in cats for specialized functions that return `Unit`, i.e. `Resource.use_`, `traverse_`, `sequence_`, `whileM_`, etc.